src/gettext.cpp

```diff
@@ -469,7 +464,6 @@ std::string dsngettext (const char * domainname, const char *singular, const cha
 	std::string msgval;
 	//Braces so that only the line that needs to be in the lock is in it:
 	{
-		std::scoped_lock lock(get_mutex());
 		msgval = bl::dngettext(domainname, singular, plural, n, get_manager().get_locale());
 	}
```

Comment:
This code should go back to `std::string msgval = bl::dngettext(domainname, singular, plural, n, get_manager().get_locale());`

Reply:
Done Sir.🫡